### ISSUE TITLE
PayPal Express isn't fully setup by default

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction#3001]
### Expected behavior
When I enable PayPal Express, I'd expect that the default settings are configured automatically.

### Actual Behavior
After enabling PayPal express, I'm forced to select when payment should be captured.

![image](https://user-images.githubusercontent.com/1203639/31093023-9c97993e-a76d-11e7-8e33-d081ede9d85e.png)

### Steps to Reproduce the Behavior
On `marketplace` branch
reset
login as admin
enable PayPal Express

### Versions
```
Node: 7.8.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.12.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.09.0-ce
```

**Comment by efalayi** [comment]
@aaronjudd what are the default settings for Paypal Express?

**Comment by zenweasel** [comment]
@efalayi I am going to make an executive decision as Vice President in charge of Paypal that the default should be "Authorize, Capture on Complete" since that's the default behavior for the rest of the payment methods.

**Comment by zenweasel** [comment]
Closed via #3027
